traffic.go

```diff
@@ -1,85 +1,147 @@
 package main
 
 import (
-	"image"
 	"image/color"
 
 	"engo.io/ecs"
 	"engo.io/engo"
-	"github.com/EngoEngine/TrafficManager/systems"
+	"engo.io/engo/common"
+	"github.com/EngoEngine/TrafficDefense/systems"
 )
 
 const (
 	KeyboardScrollSpeed = 400
 	EdgeScrollSpeed     = KeyboardScrollSpeed
 	EdgeWidth           = 20
 	ZoomSpeed           = -0.125
+
+	WorldPadding = 50
 )
 
 type myScene struct{}
 
-type HUD struct {
-	ecs.BasicEntity
-	engo.RenderComponent
-	engo.SpaceComponent
-}
-
 // Type uniquely defines your game type
 func (*myScene) Type() string { return "myGame" }
 
 // Preload is called before loading any assets from the disk, to allow you to register / queue them
 func (*myScene) Preload() {
-	engo.Files.Add("assets/textures/city.png")
+	common.AudioSystemPreload()
+	err := engo.Files.LoadMany(
+		"textures/city.png",
+		"fonts/Roboto-Regular.ttf",
+		"fonts/fontello.ttf",
+		"logic/1.level.yaml",
+		"logic/vehicles.yaml",
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	// These are allowed to fail
```

Comment:
Why is indeed something we could document. 

Well, audio makes the game better, but someone could do without. The main reason for this, is such that it doesn't `panic` in the browser, because the `.wav` files won't load there.